static/app/components/events/interfaces/replay/loader.tsx

```diff
@@ -0,0 +1,34 @@
+import {useCallback} from 'react';
+
+import ErrorBoundary from 'sentry/components/errorBoundary';
+import LazyLoad from 'sentry/components/lazyLoad';
+import type {Organization} from 'sentry/types';
+import type {Entry, EntryType, Event} from 'sentry/types/event';
+
+interface Props {
+  data: Extract<Entry, {type: EntryType.REPLAY}>['data'];
+  event: Event;
+  organization: Organization;
+  projectSlug: string;
+  isShare?: boolean;
+}
+
+function Loader({data, event, organization}: Props) {
+  const replayPreview = useCallback(
+    () => import('sentry/components/events/eventReplay/replayPreview'),
```

Comment:
Being lazy for now. 

We're going to have to refactor this preview widget to split up the data fetch from the render. And the render will need to be refactored/parameterized so we can better control the overlay stuff (or disable it all).